wiki/Tournaments/LMS/Shift/1/en.md

```diff
@@ -33,22 +33,22 @@ Taiko LMS: Shift Cup
 
  *The remaining prizes are to be decided.*
 
-## Organization
+## Organisation
```

Comment:
could be left as Organization

Reply:
Do we use `Organization` and forget about the British English @TPGPL?